src/integrationCapture.ts

```diff
@@ -180,6 +193,37 @@ export default class IntegrationCapture {
         return this.getClickIds(this.clickIds, this.filteredPartnerIdentityMappings);
     }
 
+    /**
+     * Returns only the `integration_attributes` mapped integration output.
+     * @returns {Dictionary<string>} The integration attributes.
+     */
+    public getClickIdsAsIntegrationAttributes(): IntegrationAttributes {
+        // Integration IDs are stored in the following format:
+        // {
+        //     "integration_attributes": {
+        //         "<moduleId>": {
+        //           "mappedKey": "clickIdValue"
+        //         }
+        //     }
+        // }
+        const mappedClickIds: IntegrationAttributes = {};
+
+        for (const key in this.clickIds) {
+            if (this.clickIds.hasOwnProperty(key)) {
+                const value = this.clickIds[key];
+                const mappingKey = this.filteredIntegrationAttributeMappings[key]?.mappedKey;
+                if (!isEmpty(mappingKey)) {
+                    const moduleId = this.filteredIntegrationAttributeMappings[key]?.moduleId;
+                    if (moduleId && !mappedClickIds[moduleId]) {
+                        mappedClickIds[moduleId] = { [mappingKey]: value };
+                    }
+                }
+            }
+        }
+        return mappedClickIds;
+    }
```

Comment:
is there any reason you can't use getClickIds and pass in the `filteredIntegrationAttributeMappings` as the 2nd argument like you do in lines 176-192?

Reply:
The difference is the shape of the data. For example, `getClickIds` returns a simple dictionary and therefore can be made into a simple utility function:
```
{
    clickID1: 'value',
    clickID2: 'value',
    clickID3: 'value',
}
```

But this is very specific to integration attributes, and is more of a hash map:
```
{
    "<moduleId>": {
        "clickId": "clickIdValue"
    }
}
```

I could potentially move this into a separate utility function, but it may not be necessary since we're only using it in one place. I created the previous utility function because it was being used in two different places.